manifests/b/Blueberry/FlashbackExpress/5.54.0.4702/Blueberry.FlashbackExpress.locale.en-US.yaml

```
@@ -0,0 +1,27 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Blueberry.FlashbackExpress
+PackageVersion: 5.54.0.4702
+PackageLocale: en-US
+Publisher: Blueberry Software (UK) Ltd.
+PublisherUrl: https://www.flashbackrecorder.com/
+PublisherSupportUrl: https://www.flashbackrecorder.com/support/
+PrivacyUrl: https://www.flashbackrecorder.com/privacypolicy/
+Author: Blueberry Software (UK) Ltd.
+PackageName: FlashBack Express 5
+PackageUrl: https://www.flashbackrecorder.com/express/
+License: Copyright (c) Blueberry Software (UK) Ltd. All rights reserved.
+LicenseUrl: https://www.flashbackrecorder.com/end-user-agreement-express/
+Copyright: 2003
+CopyrightUrl: https://www.flashbackrecorder.com/aboutus/
+ShortDescription: FlashBack Express 5
+Description: BB FlashBack Express 5
+Moniker: flashback.express
+Tags:
+- flashback
+- flashback express
+- flashback express 5

```

No spaces in the Tags
```suggestion
- flashback
- flashback-express
- flashback-express-5
```